master/docs/relnotes/index.rst

```diff
@@ -10,6 +10,60 @@ Release Notes
 
 .. towncrier release notes start
 
+Buildbot ``0.9.13`` ( ``2017-11-07`` )
+======================================
+
+Features
+--------
+
+- Added flag to Docker Latent Worker to always pull images
```

Comment:
Why this one feature up here with the rest below?

Reply:
arg. I edited it, but looks like I failed the commit :-(